ui/src/timeMachine/components/QueryTab.tsx

```diff
@@ -169,10 +168,9 @@ class TimeMachineQueryTab extends PureComponent<Props, State> {
     return this.props.queryCount > 1
   }
 
-  private handleRemove = (e: MouseEvent): void => {
+  private handleRemove = (): void => {
```

Comment:
The Clockface version of `RightClick` does not pass an event object back to `onClick` so I created an issue to track: https://github.com/influxdata/clockface/issues/420

Also, it looks like `stopPropagation` is already being called from within `RightClickMenuItem` so it may not even need to be called here